wsagent/che-core-api-git-shared/src/main/java/org/eclipse/che/api/git/shared/CloneRequest.java

```diff
@@ -49,6 +49,13 @@
     
     CloneRequest withRemoteName(String remoteName);
 
+    /** @return true if 'Recursive' parameter enabled */
+    boolean getRecursiveEnabled();
```

Comment:
As for me name is a little bit strange for DTO. Maybe something like isRecursive would be better